avocado/core/job.py

```diff
@@ -319,10 +319,14 @@ def _log_avocado_version():
         job_log = _TEST_LOGGER
         job_log.info('Avocado version: %s', version.VERSION)
         if os.path.exists('.git') and os.path.exists('avocado.spec'):
-            cmd = "git show --summary --pretty='%H' | head -1"
-            status, top_commit = commands.getstatusoutput(cmd)
+            cmd = "git show --summary --pretty='%H'"
+            result = process.run(cmd, ignore_status=True)
```

Comment:
@ldoktor if we shouldn't be using `avocado.utils.process` ourselves, we should remove it from our utility library.  It may have problems (code == bugs + functionality), but it's our standard way of running processes. Also, it's not the only place `process.run` is used in `avocado.core` code. 

Unless there's a blocker here that I don't seee, I'm holding on to `avocado.utils.process` usage.